jni/modules/lfo.cpp

```diff
@@ -93,17 +88,11 @@ void LFO::generate()
 
     if ( table == 0 ) {
         // no table in pool, LFO will generate table inline
-        _manageTableAllocation = true;
-        table = new WaveTable( WAVE_TABLE_PRECISION, _rate );
-        WaveGenerator::generate( table, _wave );
-        // store table inside pool
-        TablePool::setTable( table, SSTR( _wave ));
```

Comment:
OK, this was silly. If this LFO maintains its own table (see clone operation below), there's no need to temporarily store one in the TablePool as well (unless multiple LFO's will be instantiated with the same waveform, better to create and register a pool of all waveforms upfront then).